src/controllers/hid/hidcontroller.cpp

```diff
@@ -20,6 +20,11 @@ ControllerJSProxy* HidController::jsProxy() {
     return new HidControllerJSProxy(this);
 }
 
+namespace {
+constexpr int kReportIdSize = 1;             // Size of an USB HID Report ID is always one byte
+constexpr int kMaxHidErrorMessageSize = 512; // Maximum size of an HID error message string
```

Comment:
comment is redundant, constant name is clear

Reply:
I removd the comment lines